tests/test_clinvar_to_evidence_strings.py

```diff
@@ -0,0 +1,185 @@
+#         self.assertEqual(self.evidence_string, test_dict)
+
+
+class GetCttvVariantTypeTest(unittest.TestCase):
+    def setUp(self):
+        self.record_single_a = ({"reference": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACG", "alternate": "C"}, "snp single")
+        self.record_single_b = ({"reference": "A", "alternate": "C"}, "snp single")
+        self.record_single_c = ({"reference": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACG", "alternate": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACG"}, "snp single")
+
+        self.test_records_singles = [self.record_single_a, self.record_single_b, self.record_single_c]
+
+        self.record_structurals_a = ({"reference": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGT", "alternate": "C"},
+                                "structural variant")
+        self.record_structurals_b = ({"reference": "A", "alternate": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGT"},
+                                "structural variant")
+        self.record_structurals_c = ({"reference": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGT",
+                                 "alternate": "AGAGACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGTACGT"},
+                                "structural variant")
+
+        self.test_records_structurals = [self.record_structurals_a, self.record_structurals_b, self.record_structurals_c]
+
+    def test_get_cttv_variant_type_singles(self):
+        for record in self.test_records_singles:
+            self.assertEqual(clinvar_to_evidence_strings.get_cttv_variant_type(record[0]), record[1])
+
+    def test_get_cttv_variant_type_structurals(self):
+        for record in self.test_records_structurals:
+            self.assertEqual(clinvar_to_evidence_strings.get_cttv_variant_type(record[0]), record[1])
+
+
+class LoadEfoMappingTest(unittest.TestCase):
```

Comment:
Make smaller files for testing, extracts from larger file. Ensure to create a smaller ignore file too, that matches a subset of the efo mapping file.